docs/about.md

```diff
@@ -34,3 +34,22 @@ Talawa’s desired main features are:
 1. Donation acceptance
 1. Service provider level multi-organization capability
 1. Plugin support for administration services such as billing
+
+### Technologies Used
+
+Technology plays an increasingly significant role in improving access to education for people living in impoverished areas and developing countries. Some of the technologies that are being used by Talawa are:
+
+
+1. Front-end Technologies: HTML, CSS, Flutter, TypeScript and JavaScript is used to create the website's front-end user interface.
+
+2. Back-end Technologies: Talawa uses server-side scripting languages like GraphQL, React and Node.js to handle the website's back-end functionality.
```

Comment:
We don't use Node.js any more. We replaced it with Typescript. React isn't a backend technology.